pandas/core/groupby.py

```diff
@@ -2203,7 +2203,6 @@ def agg_series(self, obj, func):
     # cython aggregation
 
     _cython_functions = copy.deepcopy(BaseGrouper._cython_functions)
-    _cython_functions['aggregate'].pop('median')
```

Comment:
I ran nosetests locally, and it works.  I looked back at old versions, and I think it is a legacy issue of how the cython stuff was implemented over time.